storage.py

```diff
@@ -56,3 +57,22 @@ async def save_object(
 		)
 		obj._id = result.inserted_id
 		log.debug(f"Inserted {type(obj)} ({obj._id})")
+
+
+async def get_pokemon(
+	id: ObjectId
+) -> Pokemon: # not sure if my type annotations are correct here
+	"""Get pokemon from the database by its ObjectId.
+
+	:param id: The id of the pokemon the user is trying to retrieve
+	"""
+	coll = pokemon(
+	) # I'm not too confident I'm doign this correctly. Should I be doing these inside conditionals like in save_object?
```

Comment:
No, what would you be comparing for? You only want the pokemon collection

Reply:
Ah, good point